project/settings.py

```diff
@@ -52,6 +54,11 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+if DEBUG:
```

Comment:
I think `local_settings.py` would be a better location for this

Reply:
you mean for all the debug-related packages?

Reply:
Yes, everything after the `if DEBUG` statement, I think it would make sense. It should be documented in `README.md`